### ISSUE TITLE
Animation of arrest

### ISSUE THREAD

**Issue by jhsua** [D3jv1k/esx_policejob-with-animations#1]
The police officer is doing the animation when is arresting someone right but when you want to remove the handcuff he make the same animation, i will try to fix it by my own cause its not hard i think but i just want to make you know it :) great work to be honest!

**Comment by dawixeuu** [comment]
Hello can you contact me on discord? adyz#1562

**Comment by jhsua** [comment]
yeah sure buddy i send you a friend request my discord its like my name jhsua# etc....
